iceberg/src/main/scala/org/apache/spark/sql/delta/serverSidePlanning/SparkToIcebergExpressionConverter.scala

```diff
@@ -61,41 +59,40 @@ object SparkToIcebergExpressionConverter {
   /**
    * Convert a Spark Filter to an Iceberg Expression.
    *
-   * @param filter The Spark filter to convert
-   * @param schema The table schema (currently unused but reserved for future type conversions)
+   * @param sparkFilter The Spark filter to convert
    * @return Some(Expression) if the filter is supported, None otherwise
    */
-  def convert(filter: Filter, schema: StructType): Option[Expression] = filter match {
+  def convert(sparkFilter: Filter): Option[Expression] = sparkFilter match {
```

Comment:
do all of these convert methods need to be public? 
if not, minimize pubilc methods